components/org.wso2.carbon.identity.organization.management.application/src/main/java/org/wso2/carbon/identity/organization/management/application/listener/FragmentApplicationMgtListener.java

```diff
@@ -317,6 +318,11 @@ public boolean doPostGetServiceProvider(ServiceProvider serviceProvider, String
         return super.doPostGetServiceProvider(serviceProvider, applicationName, tenantDomain);
     }
 
+    private void inheritDiscoverabilityProperty(ServiceProvider mainApplication, ServiceProvider sharedApplication) {
```

Comment:
```suggestion
    private void configureDiscoverabilityProperty(ServiceProvider mainApplication, ServiceProvider sharedApplication) {
```
When compared to other methods ex: `inheritAPIBasedAuthenticationEnabledProperty` the inherit phrase looks wrong as it doesn't inherit the as same as other methods due to we have defined some business logic as per the limitations we have.